pkg/apis/rbac/validation/policy_comparator.go

```diff
@@ -69,9 +73,11 @@ func breakdownRule(rule rbac.PolicyRule) []rbac.PolicyRule {
 		}
 	}
 
-	// Non-resource URLs are unique because they don't combine with other policy rule fields.
+	// Non-resource URLs are unique because they only combine with verbs.
 	for _, nonResourceURL := range rule.NonResourceURLs {
```

Comment:
Yes, they have a different verb set according to our documentation[0] (with a little overlap). But unless there's a canonical list somewhere I'm more inclined to let "verbs" be anything and improve the documentation. 

[0] http://kubernetes.io/docs/admin/authorization/